src/Header/RequestParser.h

```diff
@@ -0,0 +1,24 @@
+// Copyright (c) 2021, 鍾淯丞, 周杰仕, 林仁鴻. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+#include "./OperateData.h"
```

Comment:
"./" in include is not necessary. If in doubt, set the include path in CMake.

Reply:
Include path changed